README.md

```diff
@@ -41,6 +54,18 @@ public void Configure(IApplicationBuilder app)
 }
 ```
 
+4. Optional. Implement Forte.EpiserverRedirects.Repository.IRedirectRuleRepository interface in order to use your own rule store. Specify your implementation in configuration: Step 2.
```

Comment:
It should be a little bit easier to read :D Right now it's just a single wall of text without formatting. Also there is `Step 2` but no `Step 1`. And once again, I don't see any value in presenting the code snippet below it. It does not help the reader - they should probably already now how to implement interface in C# ;)
![image](https://user-images.githubusercontent.com/84311537/205057094-3bcaa1e4-ac10-468a-82c3-6a66a077f608.png)